drivers/SmartThings/matter-hrap/src/ThreadBorderRouterManagement/server/attributes/init.lua

```diff
@@ -0,0 +1,29 @@
+local attr_mt = {}
+attr_mt.__attr_cache = {}
+attr_mt.__index = function(self, key)
+  if attr_mt.__attr_cache[key] == nil then
+    local req_loc = string.format("ThreadBorderRouterManagement.server.attributes.%s", key)
+    local raw_def = require(req_loc)
+    local cluster = rawget(self, "_cluster")
+    raw_def:set_parent_cluster(cluster)
+    attr_mt.__attr_cache[key] = raw_def
+  end
+  return attr_mt.__attr_cache[key]
+end
+
+local ThreadBorderRouterManagementServerAttributes = {}
+
+function ThreadBorderRouterManagementServerAttributes:set_parent_cluster(cluster)
+  self._cluster = cluster
+  return self
+end
+
+setmetatable(ThreadBorderRouterManagementServerAttributes, attr_mt)
+
+local has_aliases, aliases = pcall(require, "st.matter.clusters.aliases.ThreadBorderRouterManagement.server.attributes")
+if has_aliases then
```

Comment:
We can also remove all references to the aliases from the embedded cluster defs since these definitions would not use/have access to the aliases anyway